vnext/Shared/Shared.vcxitems

```diff
@@ -50,7 +50,9 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)Modules\FileReaderModule.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)Modules\HttpModule.cpp" />
     <ClCompile Include="$(MSBuildThisFileDirectory)Modules\I18nModule.cpp" />
-    <ClCompile Include="$(MSBuildThisFileDirectory)Modules\NetworkingModule.cpp" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)Modules\NetworkingModule.cpp">
+      <ExcludedFromBuild>true</ExcludedFromBuild>
```

Comment:
What does this do?  If it's not part of build, why is it here?

Reply:
It skips compilation of that file.
I'm not deleting it yet for reference purposes.
Will probably delete the file altogether by the release of 0.71.